app/models/user.rb

```diff
@@ -145,7 +145,7 @@ def self.create_blocked_scope(blocked)
   validates_uniqueness_of :login, if: Proc.new { |user| !user.login.blank? }, case_sensitive: false
   validates_uniqueness_of :mail, allow_blank: true, case_sensitive: false
   # Login must contain letters, numbers, underscores only
-  validates_format_of :login, with: /\A[a-z0-9_\-@\. ]*\z/i
+  validates_format_of :login, with: /\A[a-z0-9_\-@\.+ ]*\z/i
```

Comment:
Prefer the new style validations `validates :column, format: value` over `validates_format_of`.